src/components/member-entries/MemberEntries.tsx

```diff
@@ -23,13 +24,28 @@ export const MemberEntries = (props: MemberEntriesProps) => {
     setTeamMembers(props.teamMembers);
   }, []);
 
+  const [addTeamMember, { data }] = useMutation(CREATE_TEAM_MEMBER);
+
   async function handleClick(newMemberEntry: Types.MemberEntryProps) {
-    const formattedNewEntry = await addMemberEntry(newMemberEntry);
-    if (formattedNewEntry) {
-      setTeamMembers([...teamMembers, formattedNewEntry]);
-    }
+    addTeamMember({
```

Comment:
You could consider deconstructing the variables from the formattedNewEntry like so:  const { emailAddress, label, client, role, firstName, lastName, startingDate } = formattedNewEntry;

And then using the variables in the addTeamMember function like this: await addTeamMember({
      variables: { emailAddress, label, client, role, firstName, lastName, startingDate },
    });

This solution allows you to avoid reusing the newMemberEntry. each time. Got this advice from Ruben.